snackbars/src/main/res/values/styles-snackbar.xml

```diff
@@ -13,6 +13,15 @@
         <item name="android:paddingBottom">@dimen/vtmn_snackbars_padding_vertical</item>
```

Comment:
I would change `@dimen/vtmn_snackbars_padding_horizontal` by a value start & end because we have start: 16dp and end:8dp

<img width="935" alt="image" src="https://user-images.githubusercontent.com/46710837/236232483-0295d00d-070f-464a-8909-4b4a7169968d.png">